specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -15,39 +15,36 @@
 - [Alternatives](#alternatives)
 
 ## Summary
-
-<!--
-Short summary of the proposal. It will be used as user-focused
-documentation such as release notes or a (customer facing) development roadmap.
-The tone and content of the `Summary` section should be
-useful for a wide audience.
--->
+Create an open source frontend for the VDK.
+The frontend will let customers, create,delete deploy and generally managed nad browse their jobs.
+For frontend is written in an extensible format to easily allow other teams to build extensions.
 
```

Comment:
I am missing more information about this frontend proposal. We have a sever-client model used for data jobs management, right? What types of clients do we have currently, and why do we need another client, presumably the frontend is a client?